scripts/build

```diff
@@ -105,7 +107,12 @@ STEP_DIR=$OBJ_GCC_STATIC
 if [[ ! -d $STEP_DIR ]]; then
   printf "${WHT}Compiling GCC static${CLR}${NWL}"
 
-  rmStep $OBJ_MUSL
+  rm -rf $OBJ_MUSL
+
+  # Fix for arm builds on osx
+  if [[ $(uname -s) = "Darwin" && $ARCH = "arm" ]]; then
+    export CXXFLAGS="$CXXFLAGS -fbracket-depth=512"
+  fi
```

Comment:
Part of BigRedButton for prebuilds. Just thought I'd fix it for people in the future.